packages/marketing-modal/src/MarketingModal/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { typeScales } from '@leafygreen-ui/tokens';
+
+export const titleStyle = css`
+  margin-bottom: 4px;
+`;
+
+export const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+  overflow: hidden;
+`;
+
+export const wrapperStyle = css`
+  text-align: center;
+  padding: 0 20px 32px;
+  max-width: 476px;
+  margin: 0 auto;
+`;
+
+export const contentStyle = css`
+  font-size: ${typeScales.body1.fontSize}px;
+  line-height: ${typeScales.body1.lineHeight}px;
+`;
+
+export const contentThemeStyle: Record<Theme, string> = {
```

Comment:
i think let's be consistent with how we handle theming dark mode (called out two other instances above)